tools/cachegrand-benchmark-generator/src/main.c

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (C) 2018-2022 Vito Castellano
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the BSD license.  See the LICENSE file for details.
+ **/
+
+#include <stdio.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <string.h>
+#include <json.h>
+
+#include "all_tests.h"
+#include "matcher.h"
+#include "builder.h"
+#include "analyzer.h"
+#include "support.h"
+#include "output.h"
+
+int main(int argc, char **argv) {
+    printf("### Cachegrand Benchmark Generator ###\n");
+    char **test_lists = (char **) all_tests;
+    size_t n_tests = sizeof(all_tests) / sizeof(char*);
+
+    if (argc > 1) {
```

Comment:
Feel free to drop this, it's much better the approach of passing the list of files internally